editor/export/editor_export_platform.h

```diff
@@ -220,6 +220,8 @@ class EditorExportPlatform : public RefCounted {
 		DEBUG_FLAG_REMOTE_DEBUG_LOCALHOST = 4,
 		DEBUG_FLAG_VIEW_COLLISIONS = 8,
 		DEBUG_FLAG_VIEW_NAVIGATION = 16,
+		DEBUG_FLAG_IOS_NO_ARCHIVE = 32,
+		DEBUG_FLAG_IOS_SIMULATOR = 64,
```

Comment:
Would there be away to implement this without having iOS specific values in the generic interface?
E.g. adding some kind of threshold value for platform-specific flags which can be defined in each platform implementation? Like `DEBUG_FLAGS_PLATFORM_SPECIFIC_THRESHOLD = 16` and then in iOS code they're defined as `1 << (DEBUG_FLAGS_PLATFORM_SPECIFIC_THRESHOLD + 1)` and `1 << (DEBUG_FLAGS_PLATFORM_SPECIFIC_THRESHOLD + 2)`.

Reply:
It is not necessary to have it in debug flags, so I guess I'll change it to use internal method with the extra args, similar to `export_project_helper` in Android export.